src/Compilers/CSharp/Test/Symbol/Symbols/Source/ClsComplianceTests.cs

```diff
@@ -3804,7 +3801,7 @@ public static int Main()
         }
 
         [Fact]
-        [WorkItem(741718, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/741718")]
```

Comment:
> [WorkItem(741718, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/741718")] [](http://example.com/codeflow?start=7&length=83)

Why is this line got removed. Is the test no longer related to the work item?

Reply:
I changed this line to point to the new issue as the current `[WorkItem]` attribute points to the issue that described the prior behavior (which was a holdover from the native compiler regression suite).

Reply:
Perhaps you are changing the decision, but the work item and the test are still related in my opinion.

Reply:
I did not realize that `WorkItemAttribute` was `AllowMultiple=true`. I'll update this to keep both.